lib/Logger.ts

```diff
@@ -101,6 +106,10 @@ class Logger {
     };
   }
 
+  public static get disabledLogger(): Logger {
+    return new Logger({ disabled: true });
```

Comment:
nothing major, but I wouldn't want a getter to return a new instance on every call. So either create it only once, and return it via a getter, or simply create it as a static field (since it doesn't require any params).